src/cargo/core/workspace.rs

```diff
@@ -1245,8 +1245,9 @@ impl<'cfg> Workspace<'cfg> {
             optional_dependency_names_per_member.insert(member, optional_dependency_names_raw);
         }
 
-        let levenshtein_test =
-            |a: InternedString, b: InternedString| lev_distance(a.as_str(), b.as_str()) < 4;
+        let levenshtein_test = |a: InternedString, b: InternedString| {
```

Comment:
Since you were updating references to the algorithm, should this one also be updated?